acts_as_list.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |s|
 
 
   # Dependencies (installed via "bundle install")
-  s.add_dependency "activerecord", ">= 3.0"
+  s.add_dependency "activerecord", ">= 4.2"
+  s.add_dependency "with_advisory_lock", ">= 4.0.0"
```

Comment:
Let's be more conservative with the version string: `~> 4.0` should probably do?